common/templates/mathjax_include.html

```diff
@@ -29,8 +29,28 @@
   });
 </script>
 %endif
+<script type="text/x-mathjax-config">
+  window.HUB = MathJax.Hub;
+  MathJax.Hub.signal.Interest(function(message) {
+    if(message[0] === "End Math") {
+        set_mathjax_display_div_settings();
+    }
+  });
+</script>
+
+<script type="text/javascript">
+  function set_mathjax_display_div_settings() {
+  $('.MathJax_Display').each(function( index ) {
```

Comment:
The indentation needs to be fixed. Also you can move this function in the script tag above.